File: code_examples/4-Functions copy/8-If_Then_ElseIf_Else/CSharp/main.cs
/* Created by: Mr. Coxall
 * Created on: Sep 2020
 * This program checks a traffic light
*/

using System;

/*
 * The Program class
*/
class Program {
    static void Main() {
        // this function checks a traffic light
        string lightColor;

        // input
        Console.Write("Enter the color of the traffic light: ");
        lightColor = Console.ReadLine();

        // process and output
        if (lightColor == "red") {
            Console.WriteLine("Stop!");
        } else if (lightColor == "yellow") {
            Console.WriteLine("Slow down!");
        } else if (lightColor == "green") {
            Console.WriteLine("Go!");
        } else {
            Console.WriteLine("Not a valid color.");
        }

        Console.WriteLine("\nDone.");
    }
}

File: code_examples/4-Functions copy/8-If_Then_ElseIf_Else/JavaScript/main.js
/* Created by: Mr. Coxall
 * Created on: Sep 2020
 * This program checks a traffic light
*/

const prompt = require('prompt-sync')();

// input
const lightColor = prompt('Enter the color of the traffic light: ');

// process and output
if (lightColor === 'red') {
  console.log('Stop!');
} else if (lightColor === 'yellow') {
  console.log('Slow down!');
} else if (lightColor === 'green') {
  console.log('Go!');
} else {
  console.log('Not a valid color.');
}

console.log('\nDone.');
